docs/using-hive.md

```diff
@@ -104,15 +104,41 @@ spec:
     name: mycluster-ssh-key
 ```
 
-The hiveutil CLI (see `make hiveutil`) offers a create-cluster command for generating a cluster deployment and submitting it to the cluster defined by your current kubeconfig.
+The hiveutil CLI (see `make hiveutil`) offers a create-cluster command for generating a cluster deployment and submitting it to the Hive cluster using your current kubeconfig.
+
+To view what create-cluster generates, *without* submitting it to the API server, add `-o yaml` to the above command. If you need to make any changes not supported by create-cluster options, the output can be saved, edited, and then submitted with `oc apply`.
+
+By default this command assumes the latest Hive master CI build, and the latest OpenShift stable release. `--hive-image` can be specified to use a specific Hive image to run the install, and `--release-image` can be specified to control which OpenShift release image to install in the cluster.
```

Comment:
This is outside the scope of the changes that you are making with this PR, but `--hive-image` is no longer a supported flag. I can remove this from the docs in a separate PR.

Reply:
I'll get it.